Mathlib/Combinatorics/HalesJewett.lean

```diff
@@ -24,15 +27,20 @@ huge) finite type `ι` such that whenever `ι → α` is `κ`-colored (i.e. for
 the idea of *color focusing* and a *product argument*. See the proof of
 `Combinatorics.Line.exists_mono_in_high_dimension'` for details.
 
+*Combinatorial subspaces* are higher-dimensional analogues of combinatorial lines, defined in
+`Combinatorics.Subspace`. The extended Hales-Jewett theorem generalises the statement above from
```

Comment:
My reading of this was that subspaces are defined in `Mathlib/Combinatorics/Subspace.lean`. Maybe "defined as" instead?